### PR TITLE
feat: Add update about deposit-path specific "route rate models"

### PR DESCRIPTION
**Pull request by nicholaspai**
Allows admin to set rate models per deposit path, increasing system's capital efficiency by charging more dynamic fees

### CODE REVIEW

File: UMIPs/umip-157.md [hunk 1]
```diff
@@ -366,4 +368,4 @@ Three conditions must be met for the proposal to be deemed valid:
    at the time of proposal.
 4. No obvious griefing or manipulation of the system is being performed via this proposal.
 
-If the proposal is deemed invalid, return 0. If valid, return 1. Note: these values are scaled by `1e18`.
\ No newline at end of file
+If the proposal is deemed invalid, return 0. If valid, return 1. Note: these values are scaled by `1e18`.
```
Comment (pxrl) [inline UMIPs/umip-157.md]: Newline again? :thinking:

**Comment by pxrl** [comment]
This reads OK to me - no further comments :+1:

**Review by pxrl — COMMENTED** [review]
This reads OK to me - no further comments +1

File: UMIPs/umip-157.md (hunk 1 above)
Comment (nicholaspai) [reply UMIPs/umip-157.md]: yeah weird...

Comment (nicholaspai) [reply UMIPs/umip-157.md]: i can revert

**Review by mrice32 — APPROVED** [review]
LGTM!

**Pull request merged**